Package.swift

```diff
@@ -31,7 +31,8 @@ let package = Package(
         .package(url: "https://github.com/apple/swift-atomics.git", from: "1.2.0"),
         .package(url: "https://github.com/apple/swift-algorithms.git", from: "1.2.0"),
         .package(url: "https://github.com/StanfordBDHG/XCTRuntimeAssertions.git", from: "2.2.0"),
-        .package(url: "https://github.com/apple/swift-log", from: "1.6.0")
+        .package(url: "https://github.com/apple/swift-log", from: "1.6.0"),
+        .package(url: "https://github.com/lukaskollmer/zstd.git", branch: "lukas/fix-swift-build")
```

Comment:
TODO: switch to tagged release (or at least a revision) once https://github.com/facebook/zstd/pull/4531 is merged